README.md

```diff
@@ -13,11 +13,31 @@ Base of this code base is this [repo](https://github.com/snoyberg/file-server-de
 - [ ] upload does support multiple files
 - [ ] download path is GET /download?id=id,id1,id2&token=token
 - [ ] request to download triggers request to backend
-- [ ] download supports multiple files (zipped as one)
-- [ ] service is either mapped with a usefull prefix /userdata/ or a fake subdomain files.....de/upload...  
+- [ ] download supports multiple files (zipped as one, or as multiple downloads)
+- [ ] service is either mapped with a usefull prefix /userdata/ or /data/
 **(Roadmap feature)**
 - [ ] there is another path /preview/id?token=token
 
+
+### Upload
+- FE sends POST with /upload, files content, path and token (as Header)
+- FH sends POST with path, token and file information to rest
+- rest creates file in DB, response is 200 and ids of files
+- FH saves files in a folder (named by a hash of the file id) with filename == fileid
```

Comment:
talking about the Fh saving files. The FH is sending a post to the REST endpoint. And the endpoint returns a array of saved file instances. The instance would be a slightly changed version with a "realPath" / "realName" key that specifies how the FH needs to save the file. (when downloading the FH gets the same answer when asked for the files with the same ids.)  
( I would rewrite the endpoints in rest accordingly)

Reply:
Ok so you don't want to store the files with obfuscated names and also want to create the folder structure?

Reply:
obfuscated (which means somehow hashed) names without structure.

Reply:
Why not just use the id as filename then?

Reply:
y doesn't matter really. we are not going to implement this tomorrow